vue3-apollo-quasar/src/components/ProfilePageLayout/ProfilePageLayout.vue

```diff
@@ -86,8 +86,32 @@ function changeTab(val) {
   tab.value = val;
 }
 
+const SEARCH_QUERY = gql`
+  query Search {
+    search @client
+  }
+`;
+
+const { result: searchData, loading: loadingSearch } = useQuery(SEARCH_QUERY);
 const user = useUserStore();
 const { repos, loading } = getUserRepos(props.username, false);
+const filteredRepos = computed(() => {
```

Comment:
You can simplify this with fewer code and get same result

```js
const filteredRepos = computed(() => {
  return repos.value.filter((repo) => {
    return repo.name.match(new RegExp(searchData.value.search, 'i'));
  });
});
```

filter with return an array and with the match stuff using the flag `i` you don't have to worry about case sensitivity.

Reply:
Ah! I took that logic from a different starter project(next-react-query), and it's used in many others. Do we want to change logic away from the existing stack? As is, I don't see a reason for this to hold up the merge. 😃

Reply:
I understand. Just pointing out the simplicity and its cleaner. :smile: